solo/cli/__init__.py

```diff
@@ -137,7 +137,7 @@ def ls(all):
     for c in solos:
         descriptor = c.dev.descriptor
         if hasattr(descriptor, "serial_number") and descriptor.serial_number:
-            print(f"{descriptor.serial_number}: {descriptor.product_string}")
+            print(f"serial_number: {descriptor.serial_number}")
```

Comment:
I thought there was a product string?

Reply:
Oh yes, you're right. It's `product_name`, though. (And it's optional, at least in theory, too).
I will provide a fixed fix.

Reply:
Done.